command/rm.go

```diff
@@ -85,7 +95,10 @@ func (d Delete) Run(ctx context.Context) error {
 		return err
 	}
 
-	objChan := expandSources(ctx, client, false, srcurls...)
+	objChan := rawSource(false, srcurls...)
```

Comment:
Let's revert this raw check to make it more readable.

```go
declare objch channel
if raw {
	objch = rawSource ...
} else {
	objch, err = expandSource ..
}
```

Reply:
changed as you suggested.